http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeServiceDbQueries.scala

```diff
@@ -167,7 +166,37 @@ object RuntimeServiceDbQueries {
         .filter(_.cloudContextDb === p.asCloudContextDb)
         .filter(_.cloudProvider === p.cloudProvider)
     )
-    val runtimeQueryJoinedWithLabel = runtimeLabelQuery(clusterQueryFilteredByProject)
+
+    joinAndFilterByLabelForList(labelMap, clusterQueryFilteredByProject)
+  }
+
+  def listRuntimes(labelMap: LabelMap,
```

Comment:
can we give this one a different name...overloading functions is generally discouraged in Scala